api/app/models/spree/webhooks/validators/url_validator.rb

```diff
@@ -0,0 +1,27 @@
+module Spree
+  module Webhooks
+    module Validators
+      class UrlValidator < ActiveModel::EachValidator
```

Comment:
@sebastian-palma let's put that url validator into spree core, we will re-use it :)